README.md

```diff
@@ -15,10 +15,14 @@ Use the `--cache-directory` option to re-use previously downloaded files.
 
 ### Using AUR (Arch Linux)
 
-On Arch Linux and Arch-based operating systems, you can download package from AUR:  
+**(DANGEROUS, NOT OFFICIALLY SUPPORTED)** On Arch Linux and Arch-based operating systems, you can download package from AUR:  
 
 `yay -S gamma-launcher`  
 
+**AUR package has vulnerable (8.8 RCE) dependencie (js2py-0.74 https://nvd.nist.gov/vuln/detail/CVE-2024-28397)**
```

Comment:
AUR package related, nothing to do here. Can be removed.

However, can you put a small precision that for any question / maintenance, please address everything here: https://aur.archlinux.org/packages/gamma-launcher ? Thanks.